containers/daap-landing-to-raw/src/var/task/landing_to_raw.py

```diff
@@ -28,9 +45,34 @@ def handler(event, context):
     logger.info(f"Destination bucket: {raw_data_bucket}")
     logger.info(f"Destination key: {destination_key}")
 
-    s3.copy(
-        CopySource=copy_source,
-        Bucket=raw_data_bucket,
-        Key=destination_key,
-        ExtraArgs=s3_security_opts,
-    )
+    schema_path = config.data_product_config.schema_path(table_name=config.element.name)
+
+    inferred_schema = infer_glue_schema_from_raw_csv(
+        file_path=BucketPath(key=file_key, bucket=bucket_name),
+        data_product_element=config.element,
+        logger=logger,
+    ).metadata
+    inferred_columns = extract_columns_from_schema(inferred_schema)
+    registered_schema = read_json_from_s3(schema_path)
```

Comment:
TODO: handle errors if the schema is missing

Reply:
I think we do skip validation if there is no schema yeah